decompiler/extractor/main.cpp

```diff
@@ -16,7 +16,8 @@ enum class ExtractorErrorCode {
   VALIDATION_ELF_MISSING_FROM_DB = 4002,
   VALIDATION_BAD_ISO_CONTENTS = 4010,
   VALIDATION_INCORRECT_EXTRACTION_COUNT = 4011,
-  VALIDATION_BAD_EXTRACTION = 4020
+  VALIDATION_BAD_EXTRACTION = 4020,
+  VALIDATION_BAD_DECOMP = 4021
```

Comment:
I would change this to:
`DECOMPILATION_GENERIC_ERROR` or something.

You also need to define a useful string here https://github.com/open-goal/jak-project/blob/master/.github/scripts/releases/error-code-metadata.json